vivarium/core/emitter.py

```diff
@@ -422,7 +422,7 @@ def get_history_data_db(
     cursor = history_collection.find(experiment_query, projection)
     raw_data = []
     for document in cursor:
-        if document['data'].get('time'):
+        if document.get('assembly_id'):
```

Comment:
It doesn't look like we'll every have something in the history collection without an assembly ID:

https://github.com/vivarium-collective/vivarium-core/blob/c0966b8057221ccd8ce2c47d124b1bce240f2541/vivarium/core/emitter.py#L322-L334

Therefore, what if we asserted the presence of `assembly_id` instead? That way, if we ever run into a bug like this again, we'll get an assertion error instead of a silent failure

Reply:
done!